extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -38,7 +38,7 @@ export class VariantAnalysisManager extends DisposableObject implements VariantA
   constructor(
     private readonly ctx: ExtensionContext,
     cliServer: CodeQLCliServer,
-    storagePath: string,
+    public readonly storagePath: string,
```

Comment:
(Note: this exact same change is also in https://github.com/github/vscode-codeql/pull/1621/files#diff-9a58bafccb68e723c78ef4526185ed737f27b29330407b288eb2dbf682256332R48. I don't think it matters, we'll just see which one gets merged first 😄 )